lib/generators/shopify_app/install/templates/omniauth.rb

```diff
@@ -1,3 +1,6 @@
 # frozen_string_literal: true
+OmniAuth.config.allowed_request_methods = [:post, :get]
+OmniAuth.config.silence_get_warning = true
```

Comment:
I wonder if we should have a comment in the generated code to explain why we silence the warning here...

Reply:
👍🏻 see comment lines added

Reply:
Also should we have a migration guide somewhere in the readme for this?

Reply:
Is it possible to set these values for only `OmniAuth::Strategies::Shopify` instead of all strategies?

An app that uses multiple OmniAuth strategies may want to restrict those to only `POST` with CSRF protection.

Reply:
> Is it possible to set these values for only `OmniAuth::Strategies::Shopify` instead of all strategies?
> 
> An app that uses multiple OmniAuth strategies may want to restrict some to only `POST` with CSRF protection.

From what I can discern from https://github.com/omniauth/omniauth/blob/master/lib/omniauth/strategy.rb, I think this configuration is global only, and not on a per-provider basis.